### ISSUE TITLE
Create a mapping that upgrades the field to a multi field

### ISSUE THREAD

**Issue by heron2014** [FAC-GM/app#531]
## Investigation
Possible solution to match the right email and not create a duplicate:

```
curl -XPUT "http://dbulr/someIndex/someType/_mapping" -d '
  
   {
      "someType": {
         "properties": {
             "contacts.email": {
                 "type": "multi_field",
                  "fields": {
                     "contacts.email": {"type": "string"},
                     "original": {"type": "string", "index": "not_analyzed"}
                   }
               } 
          }
       }
    }
'
```

We told ElasticSearch that whenever it sees a property named "contacts.email" in a 'some type' document that is about to be indexed in the 'some index' index it should index it multiple times. Once into a field with the same name (contacts.email) and once into a field named "contacts.email.original" and the latter field should not be analyzed, maintaining the original value allowing is to filter by the exact contacts.email.

Then we can filter, query on the contacts.email.original 

@SimonLab what do you think?

**Comment by heron2014** [comment]
I am testing this approach on my localhost.

**Comment by heron2014** [comment]
I also tested the query on get_candidates_by_email in the csv-list :

```
"query": {
        "constant_score": {
            "filter": {
                "term": { "contact.email.original": email }
            }
        }
    }
```

But this didn't work.

**Comment by heron2014** [comment]
Maybe we need to copy the email into other property - not inside the contacts, and then reference: 

```
newEmail.original: email
```

Maybe the nested reference is a problem

**Comment by heron2014** [comment]
```
curl -XPUT "http://dbulr/someIndex/someType/_mapping" -d '

   {
      "someType": {
         "properties": {
             "contacts.email": {
                 "type": "string",
                  "fields": {
                      "raw": {"type": "string", "index": "not_analyzed"}
                   }
               } 
          }
       }
    }'
```

**Comment by heron2014** [comment]
```
curl -XPUT localhost:9200/index/type/_mapping -d '

{"properties":
   {"type": { 
      "properties": {
            "email": {"type": "string", "fields": {"original": {"type": "string", "index": "not_analyzed"}}}}}}}'
```

**Comment by heron2014** [comment]
#### Notes
After running mapping , we need to reindex the values - page 91

**Comment by heron2014** [comment]
Resolved by #536
